### ISSUE TITLE
The plugin does not load

### ISSUE THREAD

**Issue by UnEnanoMas** [JavierLeon9966/Alias#3]
```
[13:01:07] [Server thread/INFO]: Loading Alias v1.0.2
[13:01:07] [Server thread/CRITICAL]: TypeError: "Option (database.sqlite) must be of type array, was given in config.yml" (EXCEPTION) in "plugins/Alias_dev-6.phar/src/JavierLeon9966/Alias/Alias" at line 41
[13:01:07] [Server thread/CRITICAL]: #0 pmsrc/src/pocketmine/plugin/PluginManager(182): JavierLeon9966\Alias\Alias->onLoad()
[13:01:07] [Server thread/CRITICAL]: #1 pmsrc/src/pocketmine/plugin/PluginManager(346): pocketmine\plugin\PluginManager->loadPlugin(string[40] /home/container/plugins/Alias_dev-6.phar, array[2])
[13:01:07] [Server thread/CRITICAL]: #2 pmsrc/src/pocketmine/Server(1529): pocketmine\plugin\PluginManager->loadPlugins(string[24] /home/container/plugins/)
[13:01:07] [Server thread/CRITICAL]: #3 pmsrc/src/pocketmine/PocketMine(291): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[16] /home/container/, string[24] /home/container/plugins/)
[13:01:07] [Server thread/CRITICAL]: #4 pmsrc/src/pocketmine/PocketMine(321): pocketmine\server()
[13:01:07] [Server thread/CRITICAL]: #5 pmsrc(11): require(string[71] phar:///home/container/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
[13:01:07] [Server thread/CRITICAL]: Could not load plugin 'Alias'
```

```yaml
---
database:
  # The database type. "sqlite" and "mysql" are supported.
  type: sqlite

  mysql:
    host: 127.0.0.1
    # Avoid using the "root" user for security reasons.
    username: Alias
    password: mypassword123
    schema: Alias
    port: 3306
  # The maximum number of simultaneous SQL queries
  # Recommended: 1 for sqlite, 2 for MySQL. You may want to further increase this value if your MySQL connection is very slow.
  worker-limit: 1

alert: false #Everyone that has alias.alerts permission
ban: 'You are banned' #Ban message
mode: none #Options: none, ban
data: ['Address', 'ClientRandomId', 'DeviceId', 'SelfSignedId', 'XUID'] #Check for matching data in which will alert the staff members or ban the player
...
```
I have not touched anything in the configuration, it is as it comes by default

PHP: 7.4
PMMP: 3.19.0
Alias: Dev 6 (1.0.2)
Minecraft 1.16.220

**Comment by JavierLeon9966** [comment]
Add inside the database option `sqlite: []`

**Comment by UnEnanoMas** [comment]
```yaml
---
database:
  # The database type. "sqlite" and "mysql" are supported.
  type: [sqlite]

  mysql:
    host: 127.0.0.1
    # Avoid using the "root" user for security reasons.
    username: Alias
    password: mypassword123
    schema: Alias
    port: 3306
  # The maximum number of simultaneous SQL queries
  # Recommended: 1 for sqlite, 2 for MySQL. You may want to further increase this value if your MySQL connection is very slow.
  worker-limit: 1

alert: false #Everyone that has alias.alerts permission
ban: 'You are banned' #Ban message
mode: none #Options: none, ban
data: ['Address', 'ClientRandomId', 'DeviceId', 'SelfSignedId', 'XUID'] #Check for matching data in which will alert the staff members or ban the player
...
```

Didnt work:

```
[13:53:07] [Server thread/CRITICAL]: ErrorException: "Array to string conversion" (EXCEPTION) in "plugins/Alias_dev-6.phar/src/JavierLeon9966/Alias/Alias" at line 41
[13:53:07] [Server thread/CRITICAL]: #0 plugins/Alias_dev-6.phar/src/JavierLeon9966/Alias/Alias(41): pocketmine\utils\Utils::errorExceptionHandler(integer 8, string[26] Array to string conversion, string[82] phar:///home/container/plugins/Alias_dev-6.phar/src/JavierLeon9966/Alias/Alias.p, integer 41, array[3])
[13:53:07] [Server thread/CRITICAL]: #1 pmsrc/src/pocketmine/plugin/PluginManager(182): JavierLeon9966\Alias\Alias->onLoad()
[13:53:07] [Server thread/CRITICAL]: #2 pmsrc/src/pocketmine/plugin/PluginManager(346): pocketmine\plugin\PluginManager->loadPlugin(string[40] /home/container/plugins/Alias_dev-6.phar, array[2])
[13:53:07] [Server thread/CRITICAL]: #3 pmsrc/src/pocketmine/Server(1529): pocketmine\plugin\PluginManager->loadPlugins(string[24] /home/container/plugins/)
[13:53:07] [Server thread/CRITICAL]: #4 pmsrc/src/pocketmine/PocketMine(291): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[16] /home/container/, string[24] /home/container/plugins/)
[13:53:07] [Server thread/CRITICAL]: #5 pmsrc/src/pocketmine/PocketMine(321): pocketmine\server()
[13:53:07] [Server thread/CRITICAL]: #6 pmsrc(11): require(string[71] phar:///home/container/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
```

**Comment by UnEnanoMas** [comment]
```yaml
---
database:
  # The database type. "sqlite" and "mysql" are supported.
  type: [sqlite]

  mysql:
    host: 127.0.0.1
    # Avoid using the "root" user for security reasons.
    username: Alias
    password: mypassword123
    schema: Alias
    port: 3306
  # The maximum number of simultaneous SQL queries
  # Recommended: 1 for sqlite, 2 for MySQL. You may want to further increase this value if your MySQL connection is very slow.
  worker-limit: 1

alert: false #Everyone that has alias.alerts permission
ban: 'You are banned' #Ban message
mode: none #Options: none, ban
data: ['Address', 'ClientRandomId', 'DeviceId', 'SelfSignedId', 'XUID'] #Check for matching data in which will alert the staff members or ban the player
...
```
Didnt work
```
[13:53:07] [Server thread/INFO]: Loading Alias v1.0.2
[13:53:07] [Server thread/CRITICAL]: ErrorException: "Array to string conversion" (EXCEPTION) in "plugins/Alias_dev-6.phar/src/JavierLeon9966/Alias/Alias" at line 41
[13:53:07] [Server thread/CRITICAL]: #0 plugins/Alias_dev-6.phar/src/JavierLeon9966/Alias/Alias(41): pocketmine\utils\Utils::errorExceptionHandler(integer 8, string[26] Array to string conversion, string[82] phar:///home/container/plugins/Alias_dev-6.phar/src/JavierLeon9966/Alias/Alias.p, integer 41, array[3])
[13:53:07] [Server thread/CRITICAL]: #1 pmsrc/src/pocketmine/plugin/PluginManager(182): JavierLeon9966\Alias\Alias->onLoad()
[13:53:07] [Server thread/CRITICAL]: #2 pmsrc/src/pocketmine/plugin/PluginManager(346): pocketmine\plugin\PluginManager->loadPlugin(string[40] /home/container/plugins/Alias_dev-6.phar, array[2])
[13:53:07] [Server thread/CRITICAL]: #3 pmsrc/src/pocketmine/Server(1529): pocketmine\plugin\PluginManager->loadPlugins(string[24] /home/container/plugins/)
[13:53:07] [Server thread/CRITICAL]: #4 pmsrc/src/pocketmine/PocketMine(291): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[16] /home/container/, string[24] /home/container/plugins/)
[13:53:07] [Server thread/CRITICAL]: #5 pmsrc/src/pocketmine/PocketMine(321): pocketmine\server()
[13:53:07] [Server thread/CRITICAL]: #6 pmsrc(11): require(string[71] phar:///home/container/PocketMine-MP.phar/src/pocketmine/PocketMine.php)
[13:53:07] [Server thread/CRITICAL]: Could not load plugin 'Alias'
```

**Comment by JavierLeon9966** [comment]
Not that, add `sqlite: []` below the `type`, also fixed in 07de2d8
